armi/reactor/converters/axialExpansion/__init__.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2023 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Particles! Expand axially!."""
```

Comment:
Not to be a damper, but can we change this to something useful? 😄

Reply:
It would be useful to outline the contents of this package, since we are now scattering functionality across separate modules.

Reply:
lol you caught my (not so hard to find) easter egg that I dropped last summer when we were having software naming wars! 

yeah, I can lay out the contents. that's probably more useful. sigh... 😄